doc/train/multi-task-training-pt.md

```diff
@@ -0,0 +1,77 @@
+define the shared parameters of the model part {ref}`shared_dict <model/shared_dict>`, and then expand other parts for multi-model settings.
+Specifically, there are several parts that need to be modified:
+
+- {ref}`model/shared_dict <model/shared_dict>`: The parameter definition of the shared part, including various descriptors,
+type maps (or even fitting nets can be shared). Each module can be defined with a user-defined `part_key`, such as `my_descriptor`.
+The content needs to align with the corresponding definition in the single-task training model component, such as the definition of the descriptor.
+
+- {ref}`model/model_dict <model/model_dict>`: The core definition of the model part and the explanation of sharing rules,
+starting with user-defined model name keys `model_key`, such as `my_model_1`.
+Each model part needs to align with the components of the single-task training {ref}`model <model>`, but with the following sharing rules:
+- - If you want to share the current model component with other tasks, which should be part of the {ref}`model/shared_dict <model/shared_dict>`,
+you can directly fill in the corresponding `part_key`, such as
+```"descriptor": "my_descriptor", ```
+to replace the previous detailed parameters. Here, you can also specify the shared_level, such as
+```"descriptor": "my_descriptor:shared_level", ```
+and use the user-defined integer `shared_level` in the code to share the corresponding module to varying degrees
+(default is to share all parameters, i.e., `shared_level`=0).
+The parts that are exclusive to each model can be written following the previous definition.
+
+- {ref}`loss_dict <loss_dict>`: The loss settings corresponding to each task model, specified by the `model_key`.
+Each {ref}`loss_dict/model_key <loss_dict/model_key>` contains the corresponding loss settings,
+which are the same as the definition in single-task training {ref}`<loss>`.
+
+- {ref}`training/data_dict <training/data_dict>`: The data settings corresponding to each task model, specified by the `model_key`.
+Each `training/data_dict/model_key` contains the corresponding `training_data` and `validation_data` settings,
+which are the same as the definition in single-task training {ref}`training_data <training/training_data>` and {ref}`validation_data <training/validation_data>`.
+
+- (Optional) {ref}`training/model_prob <training/model_prob>`: The sampling weight settings corresponding to each `model_key`, i.e., the probability weight in the training step.
+You can specify any positive real number weight for each task. The higher the weight, the higher the probability of being sampled in each training.
+This setting is optional, and if not set, tasks will be sampled with equal weights.
+
+An example input for multi-task training two models in water system can be found here: [multi-task input on water](../../examples/water_multi_task/pytorch_example/input_torch.json).
```

Comment:
Sphinx will not render this link. You can consider one of the following
- Directly give the path, i.e. `examples/water_multi_task/pytorch_example/input_torch.json`.
- Directly show the whole file content in the documentation. See an example at https://github.com/deepmodeling/dpdispatcher/blob/ba947c05f406e439616dc5b38ddfeed250f3576b/doc/examples/template.md?plain=1#L5-L8